src/webgl/p5.Geometry.js

```diff
@@ -260,6 +266,13 @@ p5.Geometry.prototype._edgesToVertices = function() {
     dirSub.push(-1);
     this.lineNormals.push(dirAdd, dirSub, dirAdd, dirAdd, dirSub, dirSub);
     this.lineVertices.push(a, b, c, c, b, d);
+    if(data.length > 0){
```

Comment:
Super minor, but do you mind adding spaces around the brackets to match the style of the if statements elsewhere?

Reply:
It sure looks better with some space in it... ok!